src/drive/web/modules/drive/Toolbar/components/ScanWrapper.jsx

```diff
@@ -60,66 +60,69 @@ const ScanMenuItem = translate()(({ status, onClick, t, online }) => {
 class ScanWrapper extends Component {
   render() {
     const { displayedFolder, stopMediaBackup, startMediaBackup } = this.props
+    // The ActionMenu needs to stay open during the scan, so we prevent the click event from bubbling
     return (
       <MuiCozyTheme>
-        <Scanner
-          dirId={displayedFolder.id} //Pour savoir où uploader
-          pluginConfig={{
-            sourceType: 1 // Camera
-          }}
-          generateName={() => {
-            const date = new Date()
-            //We had to replace : by - since the Cordova File plugin doesn't support : in the filename
-            //https://github.com/apache/cordova-plugin-file/issues/289#issuecomment-477954331
-            return `Scan_${date.toISOString().replace(/:/g, '-')}.jpg`
-          }}
-          onConflict={'rename'}
-          //We need to cancel the MediaBackup before doing the upload since the scanned file will be
-          //inserted we don't know where in the queue resulting in a non uploaded file if the queue is
-          //big enough
-          onBeforeUpload={() => stopMediaBackup()}
-          onFinish={() => {
-            const tracker = getTracker()
-            if (tracker) {
-              tracker.push(['trackEvent', 'Drive', 'Scanner', 'Finished'])
-            }
-            startMediaBackup()
-          }}
-        >
-          {({ status, error, startScanner, filename, onClear, online }) => {
-            if (error || !filename) {
+        <div onClick={e => e.stopPropagation()}>
```

Comment:
Nit: The ScanWrapper is renderered as part of the ActionMenu hierarchy ? If so, I understand the solution here, but I think it would be cleaner to have it outside from the ActionMenu.

Reply:
>  The ScanWrapper is renderered as part of the ActionMenu hierarchy ? 

Yes, and I agree, it shouldn't be rendered elsewhere. I'd rather do it in a separate PR though, because changing this is more complicated than `ShortcutItem` for example, due to the API of `<Scanner />`.